errorHandler/errorHandler.js

```diff
@@ -0,0 +1,18 @@
+exports.psqlErrorHandler = (err, req, res, next) => {
+  console.log(err);
+  if (err?.code) {
+    res.status(400).send({ msg: "Bad Request" });
+  }
+  next(err);
```

Comment:
This needs to be an else as re.status isnt a return and it could do both of these things